Remotion/ObjectBinding/Web/Res/Themes/NovaViso/Image/sprite.svg

```diff
@@ -92,4 +92,15 @@
       <use href="#SingleChevron" />
     </g>
   </svg>
+  <svg class="sprite-part" id="RadioButtonFalse" width="16px" height="16px" viewBox="0 0 16 16">
+    <g>
+      <circle fill="#ffffff" stroke="#0c6594" cy="8" cx="8" r="7.4" />
+    </g>
+  </svg>
+  <svg class="sprite-part" id="RadioButtonTrue" width="16px" height="16px" viewBox="0 0 16 16">
+    <g>
+      <circle fill="#ffffff" stroke="#0c6594" cx="8" cy="8" r="7.4" />
+      <circle fill="#0c6594" cx="8" cy="8" r="5.5" />
+    </g>
+  </svg>
```

Comment:
```suggestion
  <svg class="sprite-part" id="RadioButtonFalse" width="16px" height="16px" viewBox="0 0 16 16">
    <g>
      <circle fill="#ffffff" stroke="#979797" stroke-width="1.1" cy="8" cx="8" r="7.4" />
    </g>
  </svg>
  <svg class="sprite-part" id="RadioButtonTrue" width="16px" height="16px" viewBox="0 0 16 16">
    <g>
      <circle fill="#ffffff" stroke="#0c6594" stroke-width="1.1" cx="8" cy="8" r="7.4" />
      <circle fill="#0c6594" cx="8" cy="8" r="5.3" />
    </g>
  </svg>
```

Some fine tuning to make it look smoother.

Also, the non-selected variant should be gray
![image](https://user-images.githubusercontent.com/6972548/134304458-8c5c77ed-314f-4af2-8e15-3e3aece7a399.png)